manifest.php

```diff
@@ -33,7 +33,7 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '13.0.0',
+    'version'     => '13.0.1',
```

Comment:
You should also now increase the dependency to `tao-core`, but since it's a huge jump you should also increase the version too: 
```
   'version' => '13.1.0'
   'requires' => array(
      //...
        'tao' => '>=17.4.0'
   )
```